src/plugins/search/search.js

```diff
@@ -277,9 +289,9 @@ export function init(config, vm) {
 
   const isExpired = localStorage.getItem(expireKey) < Date.now();
 
-  INDEXS = JSON.parse(localStorage.getItem(indexKey));
+  INDEXS = isExpired ? null : JSON.parse(localStorage.getItem(indexKey));
 
-  if (isExpired) {
+  if (!INDEXS) {
```

Comment:
When manually delete `docsify.search.index` and keep `docsify.search.expires`, this plugin can still operate normally

Reply:
The user should know what he does when he gonna manually delete index.

Reply:
> Just to reduce a manual operation step during the test, haha.

If user wanna break it, he should break it.

Reply:
Then this change is not necessary, is it?
 Let me revert this code.

Reply:
> Then this change is not necessary, is it? If not, I'll revert this code.

yep. personally, I think we do not need handle this case.

Reply:
done